actors-pekko/src/main/scala/reactivemongo/actors/util/package.scala

```diff
@@ -0,0 +1,15 @@
+package reactivemongo.actors
+
+import org.apache.pekko.util.{
+  ByteString => PekkoByteString,
+  Timeout => PekkoTimeout
+}
+
+package object util {
+  type Timeout = PekkoTimeout
+  val Timeout = PekkoTimeout
+
+  type ByteString = PekkoByteString
+  val ByteString = PekkoByteString
+
```

Comment:
```suggestion
```

Reply:
Addressed in 87d3301898f6bbf86f7167a2c141cc2bbd9e8370